tests/operator/operator.go

```diff
@@ -3442,3 +3428,15 @@ func createCdi() {
 		return err == nil && cdi.Status.Phase == sdkapi.PhaseDeployed
 	}).WithTimeout(240 * time.Second).WithPolling(1 * time.Second).Should(BeTrue())
 }
+
+func sanityCheckDeploymentsDeleted() {
+	eventuallyDeploymentNotFound("virt-api")
+	eventuallyDeploymentNotFound("virt-controller")
+}
```

Comment:
IMO this wouldn't make sense outside of this file, so it doesn't fit for a package-level function.
I suggest to either use `eventuallyDeploymentNotFound()` directly in the test or declare a lambda `sanityCheckDeploymentsDeleted()` function under the necessary scope

Reply:
@iholder101 - thanks for your comment. Here I totally diisagree.

1. there are no additional files in this package
2. there is no 3rd level of exposure in golang. Either it exposed (outside of the package) or not (only known inside the package). The "package-level" is the least exposed level. The need for test-level privacy is not the golang way. The is also no problem with "package-level" function. If there is a problem, then probably we have too large package, or we should use better naming for our functions, but I can't see the problem with moving functions to the package level.

Reply:
> @iholder101 - thanks for your comment. Here I totally diisagree.
> 
> 1. there are no additional files in this package

As I said this is not a good reasoning IMO. Files can be added in the future.

> 2. there is no 3rd level of exposure in golang. Either it exposed (outside of the package) or not (only known inside the package). The "package-level" is the least exposed level. The need for test-level privacy is not the golang way. The is also no problem with "package-level" function. If there is a problem, then probably we have too large package, or we should use better naming for our functions, but I can't see the problem with moving functions to the package level.

I agree that the fact that golang doesn't have a "private" or a "single file" level of exposure is unfortunate. I actually think that lambda functions are OK to use in such cases, since they provide a way of scoping a function.

A function such as `sanityCheckDeploymentsDeleted()` is basically not expected to be used anywhere outside this very specific context, hence I think it's better to scope it than remove the lambda which doesn't add too much value (the only thing I can think about is that it guarantees that no one will overwrite the function in runtime, which I think is unlikely and very easy to spot as opposed to simply using a function that's available in the package).

Reply:
I'm fine with opening this one for discussion. The main questions I have are:
1. How are you going to ensure, or at least make it clear, that other files should use `sanityCheckDeploymentsDeleted()`?
2. What is the value in removing the lambda in this case? How valuable is it and what is the value?
3. What's more likely - to use this function by mistake or to overwrite the lambda function variable?

Reply:
I would either drop this and use `eventuallyDeploymentNotFound` instead or keep it as a lambda function as it doesn't make sense to use outside of this file